src/test/java/com/leverx/mediator/MediatorApplicationTests.java

```diff
@@ -3,11 +3,10 @@
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
+/** @author Andrei Yahorau */
 @SpringBootTest
-class MediatorApplicationTests {
+public class MediatorApplicationTests {
 
   @Test
-  void contextLoads() {
-  }
-
+  void contextLoads() {}
 }
```

Comment:
Where are the tests?:)

Reply:
And one more comment:
Is `userId` varaible in `UserTransactionImpl` thread-safe?